File: app/src/main/java/pl/edu/pw/meil/knr/JoystickValue.java
package pl.edu.pw.meil.knr;

/**
 * Created by T460s on 09.03.2018.
 */

public class JoystickValue {

    static public int joystickX;
    static public int joystickY;
    static public FrameHandling mFrameHandling;

    static public void setX(int x) {
        joystickX = x;
    }

    static public void setY(int y) {
        joystickY = y;
    }
}

File: app/src/main/java/pl/edu/pw/meil/knr/classes/JoystickValue.kt
package pl.edu.pw.meil.knr.classes

/**
 * Created by T460s on 09.03.2018.
 */
object JoystickValue {
    var joystickX = 0
    var joystickY = 0

    fun setX(x: Int) {
        joystickX = x
    }

    fun setY(y: Int) {
        joystickY = y
    }
}